site/src/pages/identity-api-reference.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import useThemeContext from '@theme/hooks/useThemeContext';    // returns whether or not the theme is light or dark
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import { RedocStandalone } from 'redoc';
+import { lightTheme, darkTheme } from '../css/redocTheme';
+
+const RedocConfig = () => {
+  const {siteConfig} = useDocusaurusContext();
+  const {isDarkTheme} = useThemeContext();
+  return <RedocStandalone spec={siteConfig.customFields.identitySpec} options={{
+                      scrollYOffset: '.navbar', // makes the fixed sidebar and scrolling play nicey with docusaurus navbar
+                      pathInMiddlePanel: true,
+                      expandSingleSchemaField: true,
+                      menuToggle: true,
+                      theme: isDarkTheme ? darkTheme(siteConfig.customFields.bwBlue) : lightTheme(siteConfig.customFields.bwBlue)
```

Comment:
technically this can be "factored" to 
```typescript
theme: (isDarkTheme ? darkTheme : lightTheme)(siteConfig.customFields.bwBlue)
```
(equivalently):
```typescript
const themeFunc = isDarkTheme ? darkTheme : lightTheme;
//...
{
  theme: themeFunc(siteConfig.customFields.bwBlue);
}
```